apps/daimo-mobile/src/App.tsx

```diff
@@ -76,37 +78,49 @@ function AppBody() {
     (props: BottomSheetDefaultBackdropProps) => (
       <BottomSheetBackdrop
         {...props}
-        disappearsOnIndex={-1}
+        disappearsOnIndex={-0.9}
         appearsOnIndex={0}
         pressBehavior="close"
       />
     ),
     []
   );
 
+  const onChangeIndex = (index: number) => {
+    if (index === -1) {
+      setIsDebugModalOpen(false);
+    }
```

Comment:
alternately, you can remove the `setIsDebugModalOpen(true)` above and replace this function with

```
const onChangeIndex = (index: number) => setIsDebugModalOpen(index > -1)
```

Reply:
Yup, but I didn't want to rely on bottom sheet logic here, assuming it can be broken and resulting in app gestures not responding.

Wanted to create something more reliable based on simple state to control it

Reply:
But now when I'm thinking about it well, it I make it `true` and it doesn't work for real then it will be stuck in app freeze state forever ;-;

You are right it should be handled here it the same way so it rather doesn't turn on at all